### PR TITLE
Adding .github and updating repo readme

### PR DESCRIPTION
**Pull request by lomudi**
Adding `.github` with issues and PR templates.
Updating the `README.md`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,46 @@
...
+* [Full guide for creating a advertisement plugin](https://developer-zapp.applicaster.com/analytics/developing_analytics_plugins/developing_analytics_plugins.html)
+* [iOS example](./AnalyticsPlugin/iOS)
+* [Android example](./AnalyticsPlugin/Android)
+
+## login plugin
+
+Zapp enables to add different login providers to the Zapp Marketplace.
+This example include an iOS project that present the implementation of an login plugin.
+
+* [iOS example](./LoginPlugin/iOS)
```
Comment (liviur) [inline README.md:39]: Please add link to the docs:
https://developer-zapp.applicaster.com/login/intro.html

**Comment by liviur** [comment]
@lomudi - please see my comment - you can merge after adding the link.
docs are for both platforms

File: README.md [hunk 2]
```diff
@@ -1,12 +1,47 @@
-![](./assets/applicaster.png)
-
 # zapp-plugins-examples
 
 This repostory contains few `Zapp Plugins` examples that will help you to get started with creating different types of plugins for Zapp.
```
Comment (guysarfatty) [inline README.md]: a few

File: README.md [hunk 3]
```diff
@@ -1,12 +1,47 @@
...
+If you are new with developing plugins for Applicaster Zapp platform, you start with the [Getting Started Guide](https://developer-zapp.applicaster.com/getting-started/zapp-plugins.html).
```
Comment (guysarfatty) [inline README.md]: new to developing

File: README.md [hunk 4]
```diff
@@ -1,12 +1,47 @@
...
 
 ## video player plugin
+
 Zapp enables to add different players to the Zapp Marketplace.
-This example include an Andorid and iOS projects that present the implementation of the the `Zapp pluggable player API`.
+This example include an Andorid and iOS projects that present the implementation of the `Zapp pluggable player API`.
```
Comment (guysarfatty) [inline README.md]: includes

Comment (guysarfatty) [reply README.md]: remove the "an"

File: README.md [hunk 5]
```diff
@@ -1,12 +1,47 @@
...
+
+* [Full guide for creating a player plugin](https://developer-zapp.applicaster.com/player/intro.html)
+* [iOS example](./VideoPlayer/iOS)
+* [Android example](./VideoPlayer/Android)
+
+## display advertisement plugin
+
+Zapp enables to add different display advertisement plugins to the Zapp Marketplace.
+This example include an Andorid and iOS projects that present the implementation of display advertisement plugin.
```
Comment (guysarfatty) [inline README.md]: includes

Comment (guysarfatty) [reply README.md]: remove the "an"

File: README.md [hunk 6]
```diff
@@ -1,12 +1,47 @@
...
+
+* [Full guide for creating a advertisement plugin](https://developer-zapp.applicaster.com/advertisement/intro.html)
+* [iOS example](./Advertisement/iOS)
+* [Android example](./Advertisement/Android)
+
+## analytics plugin
+
+Zapp enables to add different analytics providers to the Zapp Marketplace.
+This example include an Andorid and iOS projects that present the implementation of an analytics plugin.
```
Comment (guysarfatty) [inline README.md]: includes + remove the "an"

File: README.md [hunk 7]
```diff
@@ -1,12 +1,47 @@
...
+
+* [Full guide for creating a advertisement plugin](https://developer-zapp.applicaster.com/analytics/developing_analytics_plugins/developing_analytics_plugins.html)
+* [iOS example](./AnalyticsPlugin/iOS)
+* [Android example](./AnalyticsPlugin/Android)
+
+## login plugin
+
+Zapp enables to add different login providers to the Zapp Marketplace.
+This example include an iOS project that present the implementation of an login plugin.
```
Comment (guysarfatty) [inline README.md]: includes 
presents

**Review by guysarfatty — CHANGES_REQUESTED** [review]

**Review by guysarfatty — APPROVED** [review]

**Pull request merged**